frappe/desk/search.py

```diff
@@ -10,6 +10,7 @@
 from six import string_types
 import re
 
+UNTRANSLATED_DOCTYPES = ["DocType", "Roles"]
```

Comment:
just `Role` ?

Reply:
Oh yes you are right... I'll correct it right away